### PR TITLE
added changelog from 3 -> 4

### PR DESCRIPTION
**Pull request by malaDev**
after reading both version protocol, I just wrote changes I noticed between both
If there is anything I missed, please advice

### CODE REVIEW

File: 4.0/PROTOCOL.md [hunk 1]
```diff
@@ -484,4 +484,31 @@ When built-in load balancing mechanisms are disabled, the balancing is done by t
...
+
+**REQUEST**
+- `params` now is not required anymore 
+- removed `metrics` property and added `tracing: boolean` whether tracing events is needed
+- new `paramsType: enum` needed for schema-based serializers
+- new `caller: string` property for action name of the caller
+- new `seq: int32` to have better tracking of stream request
+
+**RESPONSE**
+- new `paramsType: enum` needed for schema-based serializers
```
Comment (icebob) [inline 4.0/PROTOCOL.md]: It's not `paramsType`, instead `dataType`

**Review by icebob — CHANGES_REQUESTED** [review]
Thanks, good job!

File: 4.0/PROTOCOL.md (hunk 1 above)
Comment (malaDev) [reply 4.0/PROTOCOL.md]: oh yes

**Review by icebob — APPROVED** [review]
Thanks!

**Pull request merged**